tests/profiles-enabled/test_shipments.py

```diff
@@ -1587,17 +1593,23 @@ def test_shipment_set_shadow(self):
                 'shipmentId': shipment['id']
             }})
 
-            # Test Shipment null Device ID updates shadow
```

Comment:
Why did you remove these two test cases?

Reply:
It was interfering with later tests, so I actually just moved their tests to later and forgot to remove them